notebooks/openvino/question_answering_quantization.ipynb

```diff
@@ -0,0 +1,945 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "0407fc92-c052-47b7-8721-01836adf3b54",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-12-04T14:40:23.169301Z",
+     "iopub.status.busy": "2022-12-04T14:40:23.169096Z",
+     "iopub.status.idle": "2022-12-04T14:40:25.146758Z",
+     "shell.execute_reply": "2022-12-04T14:40:25.146281Z",
+     "shell.execute_reply.started": "2022-12-04T14:40:23.169248Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "import warnings\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import datasets\n",
+    "import evaluate\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import transformers\n",
+    "from evaluate import evaluator\n",
+    "from openvino.runtime import Core\n",
+    "from optimum.intel.openvino import OVModelForQuestionAnswering, OVQuantizer\n",
+    "from transformers import AutoModelForQuestionAnswering, AutoTokenizer, pipeline\n",
+    "\n",
+    "transformers.logging.set_verbosity_error()\n",
+    "datasets.logging.set_verbosity_error()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "889a16fe-2bc0-477e-b8d6-02a4f7508f03",
+   "metadata": {},
+   "source": [
+    "## Settings\n",
+    "\n",
+    "We define MODEL_ID and DATASET_NAME, and the paths for the quantized model files. VERSION_2_WITH_NEGATIVE should be set to TRUE if a version of the SQuAD v2 dataset is used, which includes questions that do not have an answer. \n",
+    "\n",
+    "For this tutorial, we use the [Stanford Question Answering Dataset (SQuAD)](https://huggingface.co/datasets/squad), a reading comprehension dataset consisting of questions on a set of Wikipedia articles, where the answer to every question is a segment of text from a given context. The notebook was tested with the [csarron/bert-base-uncased-squad-v1](https://huggingface.co/csarron/bert-base-uncased-squad-v1) model. Other [question-answering models](https://huggingface.co/models?dataset=dataset:squad&pipeline_tag=question-answering&sort=downloads) should also work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "c32f9a76-414b-43d9-9769-af131223f1c1",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-12-04T14:40:25.147946Z",
+     "iopub.status.busy": "2022-12-04T14:40:25.147671Z",
+     "iopub.status.idle": "2022-12-04T14:40:25.150427Z",
+     "shell.execute_reply": "2022-12-04T14:40:25.150085Z",
+     "shell.execute_reply.started": "2022-12-04T14:40:25.147933Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "MODEL_ID = \"csarron/bert-base-uncased-squad-v1\"\n",
+    "DATASET_NAME = \"squad\"\n",
+    "VERSION_2_WITH_NEGATIVE = False\n",
```

Comment:
I used the capital letters to indicate that these are "constants", to make it obvious later in the notebook that these are settings set in the beginning. It makes it much harder to accidentally use the same name in a function and then inadvertently use the global variable instead of an intended local one. 
But if this is against a style guide, I'm happy to change.

Reply:
Usually we keep lower case in our notebooks, I think capital letters might make things a little less readable but I don't have a strong opinion on this, so I'll leave this decision to you